samples/DotNet/Microsoft.Azure.ServiceBus/BasicDeadLetterQueueReceiver/Program.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace BasicDeadLetterQueueReceiver
```

Comment:
>BasicDeadLetterQueueReceiver [](start = 10, length = 28)

I wouldn't call this sample 'Basic'.
The intention is for people who are completely new to ServiceBus, they know which sample is the first sample to look at. So send/receive from a Q/Topic is a basic scenario. I would not put 'basic' in DLQ handling.